PKGBUILD/archinstall/PKGBUILD

```diff
@@ -1,33 +1,29 @@
-# Maintainer: Anton Hvornum anton@hvornum.se
-# Contributor: Anton Hvornum anton@hvornum.se
+# Maintainer: Anton Hvornum <anton@hvornum.se>
+# Contributor: demostanis worlds <demostanis@protonmail.com>
 pkgname="archinstall"
 pkgver="2.0.5"
 pkgdesc="Installs launcher scripts for archinstall"
 pkgrel=1
 url="https://github.com/Torxed/archinstall"
 license=('GPLv3')
 provides=("${pkgname}")
-md5sums=('SKIP')
 arch=('x86_64')
 source=("${pkgname}-v${pkgver}-x86_64.tar.gz::https://github.com/Torxed/archinstall/archive/v$pkgver.tar.gz")
 depends=('python-archinstall')
+md5sums=('a23ec87d901d9dc28a8a36dd7427200f')
```

Comment:
Can we change this to b2sums and/or sha512sums?

Reply:
I'd say that's a reasonable request. I skipped it mainly because I'm not the most ideal package-person, I barely got through the basic copy-paste examples hehe. but [sha256](https://wiki.archlinux.org/index.php/PKGBUILD#sha256sums) should be nice : )

Reply:
What's wrong with md5?

Reply:
Done in latest commit

Reply:
md5 has [potential collisions](https://crypto.stackexchange.com/questions/1434/are-there-two-known-strings-which-have-the-same-md5-hash-value) meaning one can sneak in stuff but get the same hash. Although unlikely, it could happen. sha256 or higher reduces that risk significantly :)